chloggen/internal/chlog/entry.go

```diff
@@ -81,6 +81,18 @@ func (e Entry) Validate(requireChangelog bool, validChangeLogs ...string) error
 		return fmt.Errorf("specify a 'component'")
 	}
 
+	// only apply prefix validation if one or more prefix is present.
+	found := !(len(componentPrefixes) > 0)
+	for _, prefix := range componentPrefixes {
+		if strings.HasPrefix(e.Component, prefix) {
+			found = true
+			break
+		}
+	}
+	if !found {
+		return fmt.Errorf("%s is not a valid 'component'. It must start with one of %v", e.Component, componentPrefixes)
+	}
```

Comment:
I find the first usage of `found` unreadable.
How about simply

```suggestion
	found := false
	for _, prefix := range componentPrefixes {
		if strings.HasPrefix(e.Component, prefix) {
			found = true
			break
		}
	}
	// only apply prefix validation if one or more prefix is present.
	if len(componentPrefixes) > 0 && !found {
		return fmt.Errorf("%s is not a valid 'component'. It must start with one of %v", e.Component, componentPrefixes)
	}
```

or

```suggestion
	// only apply prefix validation if one or more prefix is present.
	found := len(componentPrefixes) == 0
	for _, prefix := range componentPrefixes {
		if strings.HasPrefix(e.Component, prefix) {
			found = true
			break
		}
	}
	if !found {
		return fmt.Errorf("%s is not a valid 'component'. It must start with one of %v", e.Component, componentPrefixes)
	}
```